src/index.html

```diff
@@ -5,8 +5,434 @@
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="fix">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo_img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link is-active">apple</a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">samsung</a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">smartphones</a>
+          </li>
+
+          <li class="nav_list__item" data-qa="nav-hover">
+            <a
+             href="index.html"
+             class="nav_list__link"
+            >
+             laptops & computers
+            </a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">gadgets</a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">tablets</a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">photo</a>
+          </li>
+
+          <li class="nav_list__item">
+            <a href="index.html" class="nav_list__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="product-card" data-qa="card">
+        <img
+         class="product-card__img"
+         src="./images/imac.jpeg"
+         alt="iMac"
+        >
+
+        <h2 class="product-card__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">
+          Product code: 195434
+        </p>
+
+        <div class="product-card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="price">
+          <p class="price__text">
+            Price:
+          </p>
+
+          <p class="price__value">
+            $2,199
+          </p>
+        </div>
+
+        <button class="product-card__button" data-qa="hover">
+          <p>
+            buy
+          </p>
```

Comment:
Redundant, so need to remove it everywhere
```suggestion
          buy
```